community_images/haproxy/ironbank/image.yml

```diff
@@ -21,7 +21,7 @@ input_registry:
   account: ironbank
 repo_sets:
   - opensource/haproxy/haproxy24:
-      input_base_tag: "v2.4."
+      input_base_tag: "2.4."
```

Comment:
Are we sure about this?

Reply:
This is automatically updated by `prepare_ironbank_tags.sh` and the repo1 also have made same changes. i.e. removed `v` from tag and latest tag is `2.4.25`. So, its a good fix.